Bugzilla/API/V1/User.pm

```diff
@@ -0,0 +1,41 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# This Source Code Form is "Incompatible With Secondary Licenses", as
+# defined by the Mozilla Public License, v. 2.0.
+
+package Bugzilla::API::V1::User;
+
+use 5.10.1;
+use Mojo::Base qw( Mojolicious::Controller );
+use Mojo::JSON qw( true false );
+
+sub setup_routes {
+  my ($class, $r) = @_;
+  $r->get('/api/user/profile')->to('V1::User#user_profile');
```

Comment:
I know this is an existing route, but why is the `/api/` not `/rest/`?

Reply:
Because /api/user/profile is what the OAuth2 code in Phabricator currently uses. This was something Dylan set up to create a new path for Mojo implemented API calls. I like /api better than /rest but we will need to leave it as /rest for the foreseeable future. I will add another /rest/user/profile path below so that both work the same.